data.proto

```diff
@@ -32,11 +32,51 @@ message EnvironmentConfig {
   string mode = 10;
 }
 
+// Space messages are used to define environments action or observation space
+// Similar to gym's "dict" space
+message Space {
+  message Discrete {
+    int32 num = 1;
+    repeated string labels = 2; // Define labels for the discrete elements in the space, overrides `num`
+  }
+  message Box {
+    repeated uint32 shape = 1;
+    optional float low = 2; // lower bound on all dimensions, not set means unbounded
+    repeated float lows = 3; // independant lower bounds for each dimensions (gets reshaped), overrides `low`
```

Comment:
is it not sufficient to have just `lows` instead of both low and lows? If lows is None then it could imply that it's unbounded

Reply:
`low` is set the same bound on every dimension without having to repeat the value.

`lows` is there if you want to specify independent bound for each dimension